boards/common/microbit/microbit.c

```diff
@@ -137,7 +137,7 @@ static const uint8_t pixmap[ROWS][COLS] = {
 /**
  * @brief   Timer dev
  */
-#define TIMER_DEV_NUM       (1)
+#define TIMER_DEV_NUM       (2)
```

Comment:
This define is now the same as for microbit, so could be moved out of the board specific defines.

Reply:
Yes-ish -- but then actually I think this should all be moved into the respective boards' `periph_conf.h` anyway (and that change is too big for here).

That'd also then be a good place to spot which TIMER_DEVs are assigned to what.